### ISSUE TITLE
Revise main menu to support different content/layout

### ISSUE THREAD

**Issue by jonathanstegall** [MinnPost/minnpost-largo#20]
Things to consider doing:

- [x] Remove home link (can be gone entirely or kept with icon)
- [x] Put search into main menu w/ icon and scriptable form
- [x] Remove News By Region
- [x] Put Sports as a subcategory of Arts & Culture
- [x] Add Newsletters and About Us as top level items
- [x] Move Events from secondary menu to top level menu

We need to make sure that Newsletters, About Us, Events (if they're all in the main nav) do not collapse under Sections on small screens.

**Comment by jonathanstegall** [comment]
There's a small screen part, a large screen part, and a data edit part here.

- Large screen menu is in a52a7ca97c7a661094d20d2ad02d417f77fb0ca7. Still needs some JavaScript to enhance the search link.
- Primary menu data changes:
    - Move Sports under Arts & Culture
    - Remove News By Region
    - Add in this order: Newsletters (page), Events (custom link to /events), About Us (page), and Search (custom link to /?s=)
- Secondary menu data changes:
    - Remove everything except Columns and Support Us

**Comment by jonathanstegall** [comment]
Also add CSS class values (name of page is best) to home, about, newsletters, events, and search. Any top level nav item that won't be collapsed into sections is fine to do this, but the ones we need for styling/scripting are home and search.

**Comment by jonathanstegall** [comment]
There's a weird question here about how to collapse the menu.

Some options:

1. As the screen width shrinks, put stuff under a "More" dropdown. ProPublica does this, although they already have a Topics dropdown so it's not a real parallel to us. Here's [a flex pattern](https://codepen.io/kungfuyou/pen/RaJaNZ).
2. As the screen width changes, put everything under a "Menu" dropdown without a priority. This would have a more stark breakpoint. Probably similar to how it is now. But basically it would go from a full menu to no menu at all.
3. Keep the "Sections" menu and collapse the other items under something else. Maybe "More". (this is proving to be technically difficult with css.

**Comment by jonathanstegall** [comment]
With 42204d0a43946921e44af14c09865f4a3e1ba222 we have a working search box inside the nav menu.

**Comment by jonathanstegall** [comment]
With e5922da42bd1db0be52189bb8307f6653c306f67 we collapse all the items under "Menu" label. I think this is fine for now. There are, I think, open questions for how to handle all this during broader IA questions for redesign.

**Comment by jonathanstegall** [comment]
Note: we created a problem for ourselves when we kept the "featured" nav together with the main nav, because the homepage featured items are under "home" so that menu item can't be deleted; it can only be hidden.

We should attempt to address this in redesign. Partly because the featured nav is probably a bad pattern and partly because we shouldn't have a hidden menu item just so it can have child items that are displayed.
